apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -475,8 +476,11 @@ export const EventTypeList = ({
         {types.map((type, index) => {
           const embedLink = `${group.profile.slug}/${type.slug}`;
           const calLink = `${bookerUrl}/${embedLink}`;
-          const isPrivateURLEnabled = type.hashedLink?.link;
-          const placeholderHashedLink = `${WEBSITE_URL}/d/${type.hashedLink?.link}/${type.slug}`;
+          const isPrivateURLEnabled =
+            type.hashedLink && type.hashedLink.length > 0
+              ? type.hashedLink[privateLinkCopyIndices[type.slug] ?? 0]?.link
+              : "";
+          const placeholderHashedLink = `${WEBSITE_URL}/d/${isPrivateURLEnabled}/${type.slug}`;
```

Comment:
This would again not work for org domains. Please change it to the way it is being done in Advanced Tab

Reply:
@Amit91848 Done!